### PR TITLE
B16 updates

### PR DESCRIPTION
**Pull request by pieterjanstas**
Several updates in this pull request:

- Added scan 1d sawtooth class in datataker as an option
- For the laser power stabilization code, updated it to read the input and output ports from the config file instead of hardcoded. Specifically, this is done by adding the names "output" and "input" to the server dictionary in the json file: 
      {
        "type": "nidaqmx",
        "config": "laser_ctr+ai",
        "name" : "input"
      },
      {
        "type": "nidaqmx",
        "config": "b16_ao",
        "name" : "output"
      },
      {
        "type": "zi_hdawg",
        "config": "b16_hdawg"
      }
- in launcher, if there are several servers with same name, pick last server instead of choosing. (This should not really be an issue anymore now that the dangling server issue has been solved, and can also be removed honestly).
- added driver, client, server, and staticline instance for new b16 imaging laser.

### CODE REVIEW

File: pylabnet/scripts/sweeper/power.csv [hunk 1]
```diff
@@ -0,0 +1,100 @@
+0.0,1.0
```
Comment (cknaut) [inline pylabnet/scripts/sweeper/power.csv:1]: This file should probably be deleted, right?

File: pylabnet/launchers/launcher.py [hunk 2]
```diff
@@ -300,16 +300,18 @@ def _connect_matched_servers(self, matches, module, config_name, config, auto_co
                         + '                    ({})                   \n'.format(index + 1)
                         + match.summarize())
                 self.logger.info(msg_str)
-            self.logger.info('------------------------------------------\n\n'
```
Comment (cknaut) [inline pylabnet/launchers/launcher.py:303]: I think with the new fix, this change should be reverted.

**Review by cknaut — CHANGES_REQUESTED** [review]
Looks good. Would remove the `power.csv` file and revert the launcher code to the initial server picking workflow.

File: pylabnet/launchers/launcher.py (hunk 2 above)
Comment (danielra97) [reply pylabnet/launchers/launcher.py:303]: Yeah agreed

**Comment by danielra97** [comment]
Looks good other than what Can pointed out.

**Review by danielra97 — COMMENTED** [review]
Looks good other than what Can pointed out.

**Comment by pieterjanstas** [comment]
Alright I deleted the power.csv file and reverted the launcher.py code to use server picking

**Review by cknaut — APPROVED** [review]
Lgtm

**Pull request merged**